src/components/Avatar.tsx

```diff
@@ -1,10 +1,11 @@
 import { useState } from "react";
+import { Icon } from "src/components/Icon";
 import { Css, Typography } from "src/Css";
 import { useTestIds } from "src/utils";
 
 export interface AvatarProps {
   src: string | undefined;
-  name: string;
+  name?: string;
   size?: AvatarSize;
   showName?: boolean;
```

Comment:
Maybe showName can go away b/c callers can now just not pass the name prop? Or maybe we still use it for a tooltip, even when showName is false?

Reply:
Right, we could still use the name to generate the initials instead of the icon fallback. And, if we supported tooltips, then yes we'd do that too. Probably should add in that support.